fl4health/feature_alignment/tab_features_info_encoder.py

```diff
@@ -0,0 +1,194 @@
+import json
+from typing import Dict, List
+
+import pandas as pd
+from cyclops.process.feature.feature import TabularFeatures
+from flwr.common.typing import Scalar
+from sklearn.feature_extraction.text import CountVectorizer
+
+from fl4health.feature_alignment.constants import BINARY, DEFAULT_FILL_VALUES, FEATURE_TYPES, ORDINAL, STRING
+from fl4health.feature_alignment.string_columns_transformer import StringColumnTransformer
+
+
+class TargetInfoEncoder:
+    """
+    This class encodes the information about the target column
+    that is necessary to perform feature alignment.
+
+    Parameters
+    ----------
+    target: str
+        Target column.
+    target_type: str
+        Target column data type. Must be one of BINARY, NUMERICAL, ORDINAL, and STRING.
+    target_categories: List[Scalar]
+        The categories of the target column, provided it is of type ORDINAL.
+    """
+
+    def __init__(self, target: str, target_type: str, target_categories: List[Scalar]) -> None:
+        self.target = target
+        self.target_type = target_type
+        self.target_categories = target_categories
+
+    def get_target(self) -> str:
+        return self.target
+
+    def get_target_type(self) -> str:
+        return self.target_type
+
+    def get_target_categories(self) -> List[Scalar]:
```

Comment:
Since `get_target`, `get_target_type`, and `get_target_categories` are just accessor functions with no logic, I think we can just access these values directly instead of calling these functions?